docs_manual/source/ext_tool_cyberduck.rst

```diff
@@ -0,0 +1,56 @@
+.. _ext_tool_cyberduck:
+
+Cyberduck
```

Comment:
The title could be a bit more descriptive, maybe something like "Cyberduck Browser for iRODS"? Otherwise this doesn't look very informative in the index.